docs/SemanticARC.md

```diff
@@ -128,8 +145,41 @@ In order to simplify this, we will make the following changes:
    * @unowned @safe
    * @forwarding
 
+[EE] We really need a good definition of all these conventions here.
+I strongly believe that the local/global notation would help a lot for a good formal definition of the conventions.
+It's hard for the compiler to reason anything about global RCs. But local RCs are easily statically computable and for me, semantic ARC is all about dealing with local RCs.
```

Comment:
I am not talking about in the SIL syntax. I am talking about semantics.

I think that we want to only as much as we need to make our optimizations work well. That is for instance why I am considering moving RC Identity to a different proposal section (there was a reason I did not do that yet, but I don't remember, I have it written down).